zilencer/views.py

```diff
@@ -489,21 +519,34 @@ def __init__(self, reason: str) -> None:
         super().__init__(msg)
 
 
-@has_request_variables
+class RemoteServerNotificationPayload(BaseModel):
+    user_id: Optional[int] = None
+    user_uuid: Optional[str] = None
+    realm_uuid: Optional[str] = None
+    gcm_payload: Dict[str, Any] = {}
+    apns_payload: Dict[str, Any] = {}
+    gcm_options: Dict[str, Any] = {}
```

Comment:
I think ideally we'd define types for those payloads more precisely, but this isn't a regression, and it's probably not super easy, since we need to be careful to support all historical versions here... so I opened https://github.com/zulip/zulip/issues/29469 to track this.